media-api/test/lib/ElasticSearchTest.scala

```diff
@@ -58,37 +59,78 @@ class ElasticSearchTest extends FunSpec with BeforeAndAfterAll with Matchers wit
     DateTimeUtils.setCurrentMillisFixed(startDate.getMillis)
   }
 
-  describe("ES") {
+  describe("Tiered API access") {
     it("ES should return only rights acquired pictures with an allow syndication lease for a syndication tier search") {
-      val searchParams = SearchParams(tier = Syndication, uploadedBy = Some(testUser))
+      val searchParams = SearchParams(tier = Syndication)
       val searchResult = ES.search(searchParams)
       whenReady(searchResult, timeout, interval) { result =>
-        result.total shouldBe 2
+        result.total shouldBe 3
 
         val imageIds = result.hits.map(_._1)
-        imageIds.size shouldBe 2
+        imageIds.size shouldBe 3
         imageIds.contains("test-image-1") shouldBe true
         imageIds.contains("test-image-2") shouldBe true
+        imageIds.contains("test-image-3") shouldBe true
       }
     }
 
     it("ES should return all pictures for internal tier search") {
-      val searchParams = SearchParams(tier = Internal, uploadedBy = Some(testUser))
+      val searchParams = SearchParams(tier = Internal)
       val searchResult = ES.search(searchParams)
       whenReady(searchResult, timeout, interval) { result =>
         result.total shouldBe images.size
       }
     }
 
     it("ES should return all pictures for readonly tier search") {
-      val searchParams = SearchParams(tier = ReadOnly, uploadedBy = Some(testUser))
+      val searchParams = SearchParams(tier = ReadOnly)
       val searchResult = ES.search(searchParams)
       whenReady(searchResult, timeout, interval) { result =>
         result.total shouldBe images.size
       }
     }
   }
 
+  describe("syndicationStatus query") {
+    it("should return 0 results if a syndication tier specifies a SentForSyndication syndicationStatus") {
+      val search = SearchParams(tier = Syndication, syndicationStatus = Some(SentForSyndication))
+      val searchResult = ES.search(search)
+      whenReady(searchResult, timeout, interval) { result =>
+        result.total shouldBe 0
+      }
+    }
+
+    it("should return 3 results if a syndication tier specifies a QueuedForSyndication syndicationStatus") {
+      val search = SearchParams(tier = Syndication, syndicationStatus = Some(QueuedForSyndication))
+      val searchResult = ES.search(search)
+      whenReady(searchResult, timeout, interval) { result =>
+        result.total shouldBe 3
+
+        val imageIds = result.hits.map(_._1)
+        imageIds.size shouldBe 3
+        imageIds.contains("test-image-1") shouldBe true
+        imageIds.contains("test-image-2") shouldBe true
+        imageIds.contains("test-image-3") shouldBe true
+      }
+    }
+
+    it("should return 0 results if a syndication tier specifies a BlockedForSyndication syndicationStatus") {
```

Comment:
would it be useful to return a different test image (named appropriately) in each of the cases?